### ISSUE TITLE
Pg driver fails on fetching records on macOS

### ISSUE THREAD

**Issue by vrurg** [FCO/Red#145]
With a very simple model like this:

```
    model DType is table<district_type> {
         has Int $.id is serial is id;
         has Str $.type is column{ :unique };
     }
```

I cannot fetch a single record when using `Pg` driver on macOS. Any of `all`, `search`, `load` methods end up with a message similar to this one:

>```
>Too few positionals passed to 'TWEAK'; expected 2 arguments but got 0
>  in submethod TWEAK at /Users/vrurg/.rakudobrew/moar-2019.03/install/share/perl6/site/sources/5D0601150F79C8C2933047792DE54FF88C0D0F81 (Red::ResultSeq::Iterator) line 12
>  in method iterator at /Users/vrurg/.rakudobrew/moar-2019.03/install/share/perl6/site/sources/A16B6D7529C7B5072F6F9C91A58517986F2FD789 (Red::ResultSeq) line 39
>  in method cache at /Users/vrurg/.rakudobrew/moar-2019.03/install/share/perl6/site/sources/A16B6D7529C7B5072F6F9C91A58517986F2FD789 (Red::ResultSeq) line 32
>  in block <unit> at test-red.p6 line 18
>```

I saw an open issue related to `epoll`, but this one doesn't look like it's related somehow.

**Comment by vrurg** [comment]
Ok, this one was really easy to fix: #146

**Comment by FCO** [comment]
Thanks!
